docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,45 @@
+
+= Scheduled Outages
+
+Scheduled Outages allow for suppressing notifications and pausing Status Polling, Threshold Checking, and Data Collection in {page-component-title}.
```

Comment:
```suggestion
Scheduled outages let you suppress notifications and pause status polling, threshold checking, and data collection during a specified time period.
```

Reply:
Let's add an additional sentence as to why someone would do this.

Reply:
Examples can be:

* Updating components, scheduled reboots
* planned decommissioning of nodes
* also outside influences like, power-, internet-, or data center provider are working on their products
* avoid sending notifications to not spam Slack/Email channels
* don't trigger on-call guys
* regular infrastructure test